flink-connectors/flink-connector-elasticsearch-base/pom.xml

```diff
@@ -37,7 +37,7 @@ under the License.
 
 	<!-- Allow users to pass custom connector versions -->
 	<properties>
-		<elasticsearch.version>5.1.2</elasticsearch.version>
+		<elasticsearch.version>5.3.3</elasticsearch.version>
```

Comment:
Just a question. Why not the newest 5.x version?

Reply:
5.3 is the lowest version that has the cgroups issue fixed. 5.3.3 is the highest version in the 5.3.x family.
I wanted to pick the lowest possible 5.x release to be compatible with as many 5.3+ releases as possible.